tests/test_group.py

```diff
@@ -18,7 +19,7 @@ def test_group_description_edit(self, base_url, selenium, vouched_user):
         home_page.login(vouched_user['email'], vouched_user['password'])
 
         # Create a new group
-        group_name = (time.strftime('%x-%X'))
+        group_name = (uuid.uuid4())
```

Comment:
This is failing for me with `TypeError: object of type 'UUID' has no len()` is it passing for you?

Full trace:
```
self = <tests.test_group.TestGroup instance at 0x7fdca377aea8>
base_url = 'https://mozillians-dev.allizom.org'
selenium = <selenium.webdriver.remote.webdriver.WebDriver (session="614b2867a8c64a3485849e50b6629aed")>
vouched_user = {'email': '77ae49b1-364e-401b-9c38-8f38a02a6755@restmail.net', 'name': '77ae49b1', 'password': '1f72a184-bc07-4e52-a8f4-2f0da91c291c'}

    @pytest.mark.credentials
    def test_group_type_change(self, base_url, selenium, vouched_user):
        home_page = Home(base_url, selenium)
        home_page.login(vouched_user['email'], vouched_user['password'])
   
        # Create a new group
        group_name = (uuid.uuid4())
        settings = home_page.header.click_settings_menu_item()
>       group = settings.create_group(group_name)

tests/test_group.py:68:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
pages/settings.py:67: in create_group
    create_group.create_group_name(group_name)
pages/create_group_page.py:20: in create_group_name
    element.send_keys(group_name)
.tox/tests/lib/python2.7/site-packages/selenium/webdriver/remote/webelement.py:343: in send_keys
    local_file = self.parent.file_detector.is_local_file(*value)
.tox/tests/lib/python2.7/site-packages/selenium/webdriver/remote/file_detector.py:48: in is_local_file
    file_path = ''.join(keys_to_typing(keys))
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

value = (UUID('3073eb64-3b51-4f44-b806-c1934d3c00b1'),)

    def keys_to_typing(value):
        """Processes the values that will be typed in the element."""
        typing = []
        for val in value:
            if isinstance(val, Keys):
                typing.append(val)
            elif isinstance(val, int):
                val = str(val)
                for i in range(len(val)):
                    typing.append(val[i])
            else:
>               for i in range(len(val)):
E               TypeError: object of type 'UUID' has no len()

.tox/tests/lib/python2.7/site-packages/selenium/webdriver/common/utils.py:150: TypeError
```